tests/StreamWrapperTest.php

```diff
@@ -66,6 +66,17 @@ public function testStreamContext()
         $this->assertEquals('foo', file_get_contents('guzzle://stream', false, StreamWrapper::createStreamContext($stream)));
     }
 
+    public function testStreamCast()
+    {
+        $streams = [
+            StreamWrapper::getResource(Psr7\stream_for('foo')),
+            StreamWrapper::getResource(Psr7\stream_for('bar'))
+        ];
+        $write = null;
+        $except = null;
```

Comment:
those variables are unused

Reply:
`stream_select` does not accept values passed by value. Variable must be used:
http://php.net/manual/en/function.stream-select.php
If you try this:
```
stream_select($streams, null, null, 0);
```
you will get the following error: `Cannot pass parameter 2 by reference`